test/framework/ownerreference_helpers.go

```diff
@@ -382,6 +382,22 @@ func setClusterPause(ctx context.Context, cli client.Client, clusterKey types.Na
 	Expect(cli.Patch(ctx, cluster, pausePatch)).To(Succeed())
 }
 
+// SkipClusterObjectsWithoutNameContains is used in e2e tests where the owner graph
```

Comment:
I wonder if we should move this func to ownergraph.go

It seems to make a lot of sense there and here within "ownerreference_helpers.go" it fells a bit misplaced